include/my_sdnotify.h

```diff
@@ -0,0 +1,17 @@
+
+#ifndef _my_sdnotify_h
+#define _my_sdnotify_h
```

Comment:
According to coding style, please use MY_SD_NOTIFY_INCLUDED instead of _my_sdnotify_h.